src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/security/VideoBookingAccess.kt

```diff
@@ -23,6 +26,8 @@ fun checkVideoBookingAccess(externalUser: User, booking: VideoBooking) {
 }
 
 // Unknown courts or probation teams cannot be accessed by external users, they will only ever be set up by prison users
-private fun VideoBooking.isAccessibleBy(user: User) = (user.isCourtUser && court?.isUnknown() == false) || (user.isProbationUser && probationTeam?.isUnknown() == false)
+private fun VideoBooking.isAccessibleBy(user: ExternalUser) =
+  (user.isCourtUser && court?.isUnknown() == false && user.hasAccessTo(court)) ||
```

Comment:
Yes I know the `unknown` check is still here, it can't slip through by mistake keeping the check here.